packages/transit-vehicle-overlay/src/VehicleGeometry/index.js

```diff
@@ -1,8 +1,11 @@
 import React from "react";
 import PropTypes from "prop-types";
 import { FeatureGroup } from "react-leaflet";
-import { leafletPathType } from "@opentripplanner/core-utils/lib/types";
-import { vehicleType } from "../types";
+import {
+  leafletPathType,
+  transitVehicleType
+} from "@opentripplanner/core-utils/src/types";
```

Comment:
This should be imported from

>  @opentripplanner/core-utils/**lib**/types

Reply:
Hey @evansiroky  ... I think we want to be importing the /src/ code, since importing the lib 
(transpilled) code prevents webpack from a latter tree-shake when an app does a production build.  See #65

Reply:
BTW, will be good to talk thru this in an upcoming meeting/chat.